code/modules/research/devicecrafting/obj_stuff.dm

```diff
@@ -0,0 +1,29 @@
+/obj
```

Comment:
why does every single object need to hold devices for crafting?

They're just creating the final device in a frame? why is this so high in the typepath?